### ISSUE TITLE
ValueError when loading a model saved with its optimizer with custom loss function

### ISSUE THREAD

**Issue by eliphatfs** [keras-team/keras#8210]
```python
def loss_softmax_cross_entropy_with_logits(y_true, y_pred):
    return tf.nn.softmax_cross_entropy_with_logits(labels=y_true,
                                                   logits=y_pred)
```

**Comment by eliphatfs** [comment]
I'm sorry but it seemed that I created the incomplete issue a long time ago and forgot it 🤣
